bids-validator/validators/bids/checkDatasetDescription.js

```diff
@@ -33,6 +34,18 @@ const checkDatasetDescription = (jsonContentsDict) => {
   return issues
 }
 
+const checkNameField = (name) => {
+  const issues = []
+  // missing name will be caught by validation (later)
+  if (name !== null) {
```

Comment:
```suggestion
  if (name !== undefined) {
```
Name will be `undefined` here if it is missing from the datasetDescription object.